docker/puppetdb/docker-entrypoint.sh

```diff
@@ -11,10 +11,10 @@ if [ ! -d "/etc/puppetlabs/puppetdb/ssl" ] && [ "$USE_PUPPETSERVER" = true ]; th
     sleep 1
   done
   set -e
-  /opt/puppetlabs/bin/puppet config set certname "$HOSTNAME"
-  /opt/puppetlabs/bin/puppet config set server "$PUPPETSERVER_HOSTNAME"
-  /opt/puppetlabs/bin/puppet agent --verbose --onetime --no-daemonize --waitforcert 120
-  /opt/puppetlabs/server/bin/puppetdb ssl-setup -f
+  /ssl.sh
+  /ssl-setup.sh -f
 fi
 
-exec /opt/puppetlabs/server/bin/puppetdb "$@"
+# exec /opt/puppetlabs/server/bin/puppetdb "$@"
+exec java -cp /puppetdb.jar clojure.main -m puppetlabs.puppetdb.core "$@" \
+    -c /etc/puppetlabs/puppetdb/conf.d/
```

Comment:
You changed the entrypoint for the main puppetdb container but didn't include the new files in the non-slim dockerfile which breaks things

Reply:
Thanks.

What's broken?

I don't see any changes to the entrypoint script?

I don't follow.

Reply:
You edited the entrypoint in the `docker/puppetdb` directory to call the new ssl scripts and execute the jar, but the existing puppetdb containers install from packages and don't have the new ssl scripts so if I do a `make build` and a `make test` the tests fail (the non-alpine puppetdb container won't start)

Reply:
Oh. I hadn't addressed CI and the make stuff yet, figured I would do that once the container changes were all done. That's the "WIP" part 😄

Reply:
Ok started working through the CI/make stuff....

Got "make lint" and "make build" working, but "make test" fails miserably as the tests are tightly-coupled to the old package-based approach.

We'll probably have to re-think what tests make sense in this new puppetdb-from-source container rather than "fixing" the current tests.